app/javascript/mastodon/features/notifications/components/column_settings.js

```diff
@@ -42,6 +43,13 @@ export default class ColumnSettings extends React.PureComponent {
           </div>
         )}
 
+        {alertsEnabled && browserSupport && browserPermission === 'default' && (
+          <div className='column-settings__row column-settings__row--with-margin'>
+            <span className='warning-hint'><FormattedMessage id='notifications.permission_required' defaultMessage='Desktop notifications are unavailable because the required permission has not been granted.' /></span>
+            <GrantPermissionButton onClick={onRequestNotificationPermission} />
```

Comment:
There needs to be margin between the line above and the button.

Reply:
I wanted it to be visually grouped with the text.

![image](https://user-images.githubusercontent.com/384364/102204789-95a70900-3eca-11eb-93b7-a2affc2682b6.png)

Would this be better?

![image](https://user-images.githubusercontent.com/384364/102204991-dbfc6800-3eca-11eb-8bea-ab82fd0f1793.png)

Reply:
Okay, with the differnet margin to the button below, it's not worse but not better either. If you want it to be visually grouped, why not make the "button" have the same text color and maybe even make it inline part of the text like a link, underlined?

Reply:
Something like this?

![image](https://user-images.githubusercontent.com/384364/102241346-272d6f80-3ef9-11eb-9f8f-2862fb00647c.png)

Reply:
Looks nice!